product-service/src/types/types.ts

```diff
@@ -5,9 +5,11 @@ export interface Product {
   price: number;
 }
 
-export interface PostParams {
+export interface InputProduct {
   title: string;
   description?: string;
   price: number;
   count: number;
 }
+
+export const REGION = 'eu-west-1';
```

Comment:
by meaning REGION constant should be in some config module, not in types

Reply:
True. I think, even better to move in env file